README.md

```diff
@@ -27,6 +27,7 @@ segmentation map refers to pixels in the resized image, not the original input i
 The model files are hosted on IBM Cloud Object Storage. The code in this repository deploys the model as a web service
 in a Docker container. This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/).
 
+TODO - add info about building mobile vs full
```

Comment:
So does `docker build -t model-name mobile.Dockerfile` build the mobile version? 

We should just check how Dockerhub is going to pick up the different docker file builds. I think it may require the mobile Dockerfile to be in a subfolder, as in `mobile/Dockerfile`

Reply:
I think `docker build -t model-name -f mobile.Dockerfile` is the correct way to do it but it may work without the `-f`.

If dockerhub wants the file to be in a sub folder than we can do that.

I have mixed feelings about this PR. I understand the concern about the image size but this doesn't feel like an elegant solution.